dev-docs/bidders/roundhouseads.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: bidder
+title: RoundhouseAds
+description: RoundhouseAds Prebid Bidder Adapter
+biddercode: roundhouseads
+tcfeu_supported: false
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+media_types: banner, video, native
+floors_supported: true
+pbjs: true
+sidebarType: 1
+---
+
+### Overview
+
+The RoundhouseAds Bidder Adapter connects to RoundhouseAds' advertising platform, enabling bids for display, video, and native ad formats. Please contact: [info@roundhouseads.com](mailto:info@roundhouseads.com) to get started.
+
+### Bid Parameters
+
+{: .table .table-bordered .table-striped }
+| Name           | Scope    | Description                                             | Example                | Type    |
+|----------------|----------|---------------------------------------------------------|------------------------|---------|
+| `placementId`  | required | Unique placement identifier provided by RoundhouseAds    | `'12345'`              | string  |
+| `video`        | optional | Video-specific settings (only for video ad units)       | `{ language: 'en' }`   | object  |
+
+### Test Parameters
+
+To test the RoundhouseAds adapter, use the following example ad units:
+
+```javascript
+var adUnits = [
+  {
+    code: 'banner-ad-unit',
+    mediaTypes: {
+      banner: {
+       sizes: [[300, 250]]
+      }
+    },
+    bids: [
+      {
+       bidder: 'roundhouseads',
+       params: {
+         placementId: 'your-placement-id'
+        }
+      }
+    ]
+  },
+  {
+    code: 'native-ad-unit',
+    mediaTypes: {
+      native: {
+        title: { required: true },
```

Comment:
I would recommend using the OpenRTB 2.5 native configuration instead.
The custom prebid configuration may be removed at some point in time